packages/sdk/package.json

```diff
@@ -1,63 +1,73 @@
 {
-	"name": "@buildwithsygma/sygma-sdk-core",
-	"version": "1.1.0",
-	"description": "Core primitives for bridging and message passing",
-	"main": "dist/src/index.js",
-	"types": "dist/src/index.d.ts",
+  "name": "@buildwithsygma/sygma-sdk-core",
+  "version": "1.1.0",
+  "description": "Core primitives for bridging and message passing",
+  "main": "./dist-cjs/index.js",
+  "module": "./dist-esm/index.js",
+  "source": "./src/index.ts",
+  "types": "./types/src/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist-esm/index.js",
+      "require": "./dist-cjs/index.js",
+      "types": "./types/src/index.d.ts"
+    }
+  },
```

Comment:
How sure are you that is configured correctly?
(Asking because I know is a pain in the ass to do it correctly)

Reply:
The only issue that I have is with CRA. But that is because they use outdated webpack config. That's why I've updated all examples that are based on CRA

Reply:
The issue being not related to webpack as a standalone tool, but rather pertains to the default configuration of webpack provided by CRA. It should be noted that our examples and UI do not utilize the default configuration of CRA, but instead employ `craco` and `react-app-rewired` due to the need for [node polyfills and etc.](https://github.com/sygmaprotocol/sygma-sdk/blob/main/examples/erc20-react-example/config-overrides.js)